site/guide/configuration/register-with-validmind.qmd

```diff
@@ -0,0 +1,222 @@
+:::
+
+:::: {.flex .flex-wrap .justify-around}
+::: {.w-40-ns}
+![ValidMind's main dashboard](main-dashboard.png){fig-alt="Screenshot of the main ValidMind dashboard"}
+:::
+::: {.w-40-ns}
+![Sample models populated by ValidMind](sample-models.png){fig-alt="Screenshot of the sample models populated by ValidMind"}
+:::
+::::
+
+## Accept an invite 
+
+::: {.callout title="This method will allow you to register as user linked to your institution's existing organization."}
+These steps require that you have been invited[^12] by a Customer Admin user to join your organization on ValidMind.
+:::
+
+1. Check the email inbox for the address you provided your organization to be associated with ValidMind for the following email:
+
+    - From: `no-reply@validmind.ai`
+    - Subject: "{User} invited you to join ValidDemo on ValidMind"
+
+    `{User}` is replaced with the name of the Customer Admin or other user who invited you to join. 
+
+::: {.column-margin}
+If you cannot locate this email, you may need to review your spam or junk mail folders and filters, or request that your IT allow-list our owned domains or retrieve any messages sent by:
+
+a. `validmind.ai`
+b. `validmind.com`
+:::
+
+2. Within the email, click **Accept Invitation**.
+
+![ValidMind organization invite email](organization-invitation-email.png){width=80% fig-alt="Screenshot of the Validmind organization invite email"}
+
+3. On the page that opens up in your web browser, click **Accept Invitation**.
+
+![ValidMind accept invitation to join an organization screen](organization-accept-invitation.png){width=80% fig-alt="Screenshot of the ValidMind accept invitation to join an organization screen"}
+
+::: {.column-margin}
+![ValidMind signup welcome screen](signup-welcome-screen.png){width=90% fig-alt="Screenshot of the Validmind signup welcome screen"}
+:::
+
+4. On the Welcome screen, enter your email address and input a secure, unique password, or choose one of the single sign-on options to link to your account:
+
+    - **Google**[^13]
+    - **GitHub**[^14]
+    - **Microsoft**[^15]
+
+::: {.callout-important}
+Make sure that the email or account you use to link to ValidMind is the one that was invited to join your organization.
+:::
+
+5. Check the email inbox for the address you used to sign up with ValidMind for the following email:
+
+    - From: `no-reply@validmind.ai`
+    - Subject: "Verify your email"
+
+![Verification email sent by ValidMind](verify-email.png){width=80% fig-alt="Screenshot of the verification email sent by ValidMind"}
+
+  
+::: {.column-margin}
+If you cannot locate this email, ou may need to review your spam or junk mail folders and filters, or request that your IT allow-list our owned domains or retrieve any messages sent by:
+
+a. `validmind.ai`
+b. `validmind.com`
+:::
+
+6. Click **<u>Confirm my account</u>** to gain full access to ValidMind.
+
+    - Return to the main ValidMind dashboard[^16] by clicking on **Back to ValidMind**. 
```

Comment:
Line 159 and 160 feel more like they belong into "What's next" and should not be bullets under 6. Alternatively, they could be regular text that appears after step 6. without any indentation.